bigtable/google/cloud/bigtable/row_data.py

```diff
@@ -85,6 +85,13 @@ def __eq__(self, other):
 
     def __ne__(self, other):
         return not self == other
+    
+    def __repr__(self):
+        return "<{name} value={value!r} timestamp={timestamp}>".format(
+            name=self.__class__.__name__,
+            value=self.value,
+            timestamp=self.timestamp
+        )
```

Comment:
black wants to format this all on one line, e.g.:

```python
    def __repr__(self):
        return "<{name} value={value!r} timestamp={timestamp}>".format(
            name=self.__class__.__name__, value=self.value, timestamp=self.timestamp
        )
```

Reply:
I made this PR in github's code editor, shame on me! I believe I've fixed both of the linting/black issues now, but I'm still only in github's code edtior, so I might be proven wrong!